test/e2e/upgradetest/framework/framework.go

```diff
@@ -77,6 +80,9 @@ func (f *Framework) CreateOperator() error {
 			Namespace: f.KubeNS,
 		},
 		Spec: appsv1beta1.DeploymentSpec{
+			Strategy: appsv1beta1.DeploymentStrategy{
+				Type: appsv1beta1.RecreateDeploymentStrategyType,
```

Comment:
@hongchaodeng When the strategy type is rolling update, with `maxUnavailable=1` and number of replicas is 1, it almost means the same thing as RecreateDeploymentStrategy. Since the old pod will necessarily get terminated before the new one can become ready(with our implementation of readyz). So I just thought it was cleaner to use Recreate.

However since it works for `maxUnavailable=1` as well I'm going to change it to that and stress test again.